tensorboard/components/tf_backend/test/backendTests.ts

```diff
@@ -105,6 +105,20 @@ describe('backend tests', () => {
               '/data/plugin/scalars/scalar');
         });
 
+        it('ignores leading slash in dataDir', function() {
```

Comment:
These tests logically are more testing `createRouter()` than `pluginRoute()`, and seem like either way they should be grouped with the existing test for "removes trailing slash from base route".

Can you also add an explicit test for the case where it adds a leading slash when there isn't one there?  This is implicitly tested by the "removes trailing slash" test but it'd be better to have an explicit test.  Or it could be merged with the test here as something like "ensures at least one leading slash in dataDir".